docs/document/content/features/encrypt/limitations.cn.md

```diff
@@ -4,6 +4,7 @@ weight = 2
 +++
 
 - 需自行处理数据库中原始的存量数据；
+- 模糊查询支持%、_，暂不支持escape；
```

Comment:
Please add a space between chinese characters and symbols(or english word).

Reply:
ok